index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-setup-and-mobile-first</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="menu">
+        <a href="#">My Logo</a>
+        <div class="menu-bar">
+          <div id="menu-bar-1"></div>
+          <div id="menu-bar-2"></div>
+          <div id="menu-bar-3"></div>
+        </div>
+      </nav>
+      <section class="main-section">
+        <div class="content">
+          <div class="heading">
+            <h2>I’m Muhammed Glad to see you!</h2>
```

Comment:
- Kindly, your main title is an `h1` tag. for semantic and search engines. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. 👍👍